src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,21 @@
+$day-size: 100px;
+$border-color: #000;
```

Comment:
following good practice color variables should start with name $color, so please rename your variables pointing color to be, e.g:

```
$color-border
$color-day-background-hover
```

etc.